cl/lib/es_signal_processor.py

```diff
@@ -715,6 +729,39 @@ def handle_reverse_actions(self, sender, instance=None, **kwargs):
                 affected_fields,
             )
 
+    @elasticsearch_enabled
+    def handle_bulk_create(self, sender, instances=None, **kwargs):
+        """Receiver function that gets called after a bulk_create_with_signal
+        action.
+        """
+
+        if not instances:
+            return
+        instance = instances[0]
+        match instance:
+            case OpinionsCitedByRECAPDocument() if self.es_document is ESRECAPDocument:  # type: ignore
+                if settings.ELASTICSEARCH_RECAP_CITES_DISABLED:
```

Comment:
One thought: We should move towards every variable being `ENABLED` instead of `DISABLED` so that when we're looking at our settings it's not a mish-mash of Trues and Falses. I think ENABLED = True makes more intuitive sense than DISABLED=False, so let's standardize on that whenever we can.

Reply:
Sure I've changed the setting to `ELASTICSEARCH_RECAP_CITES_ENABLED`